### DISCUSSION TITLE
WebTransport msg/sec < WebSocket msg/sec ?

### DISCUSSION THREAD

**Discussion by matthiasgeihs** [socketio/socket.io#4903] [Q&A] [answered]
Was going through the [WebTransport example](https://socket.io/get-started/webtransport).
Edited this slightly to benchmark message throughput performance.
To my surprise, WebTransport can handle less messages than Websocket (50k vs 80k msg/sec).

Is this to be expected?

**Comment by darrachequesne** [comment, marked as answer]
Hi! In theory, WebTransport should be more performant.

However, the current implementation is younger than the battle-tested `ws` package, so that could explain the difference.
